src/SpendPermissionManager.sol

```diff
@@ -450,10 +455,8 @@ contract SpendPermissionManager is EIP712 {
             revert SpendValueWithdrawAmountMismatch(value, withdrawRequest.amount);
         }
 
-        // Get the hash of the spend permission
+        // check WithdrawRequest nonce suffix matches spend permission hash suffix.
```

Comment:
double check my language here please @amiecorso!

Reply:
this is great, much clearer than "hash portion matches expected" haha